solidity/contracts/OngoingAirdrops.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import './utils/Governable.sol';
+import '../interfaces/IOngoingAirdrops.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/utils/cryptography/MerkleProof.sol';
+
+abstract contract OngoingAirdrops is Governable, IOngoingAirdrops {
```

Comment:
Why abstract? How do you imagine this will be used?

Reply:
Forgot to take it out.